pkg/snapshot/snapshotter/snapshotter.go

```diff
@@ -388,7 +388,7 @@ func (ssr *Snapshotter) TakeDeltaSnapshot() (*brtypes.Snapshot, error) {
 		return nil, fmt.Errorf("failed to get compressionSuffix: %v", err)
 	}
 	snap := snapstore.NewSnapshot(brtypes.SnapshotKindDelta, ssr.prevSnapshot.LastRevision+1, ssr.lastEventRevision, compressionSuffix)
-	snap.SnapDir = ssr.prevSnapshot.SnapDir
+	snap.SnapDir = ""
```

Comment:
```suggestion
```
`SnapDir` is already `""` as returned by `NewSnapshot`.

Reply:
Right, I knew. But I tried to be extra sure! in case, some code overwrite it (Though I can't imagine what can cause it). Are you sure we should remove the line?

Reply:
If such a possibility is concrete then we should have a test case. But if we cannot yet think of a way how it might happen, let's not add addition code to handle it.